mms/tests/integration_tests/onnx_mxnet_export_multiple_model_integ_test.py

```diff
@@ -0,0 +1,34 @@
+import json
```

Comment:
this whole test seems redundant. what does it do that is unique?

Reply:
@jesterhazy  This tests multiple models from onnx-modelzoo together. Should we have only one tests for multiple models or one for onnx and one for mxnet?